### DISCUSSION TITLE
Simulation for population genetics Shiny App

### DISCUSSION THREAD

**Discussion by JoseBlanca** [posit-dev/py-shiny#1402] [Show and tell]
I'd just like to thank you for your efforts in creating shiny. I'm using the Python version to create simulations for my population genetics students.
First I tried jupyterlite, but I think shiny is a better fit for this kind of application.
I started with shiny last week and this is just a first rough version, but I think is going to work for me:

https://bioinf.comav.upv.es/pop_lab/

The Python code is in my github:

https://github.com/JoseBlanca/pop_lab

Best,

Jose Blanca

P. S. I'm using shinylive, so I can deploy in a static web server. Neat.

**Comment by gadenbuie** [comment]
That's very cool, thanks for sharing! Is the GitHub repo private?

**Comment by JoseBlanca** [comment]
Sorry, I forgot that the repository was private. Now is public.

My intention is to create some applications that could be used by any population genetic teacher, and I plan to release any code under an open source license.

shinylive is great because I can deploy in githubio and I'm sure that the applications will be always available and that I don't need to be worried by the server computing costs.

Before I used jupyterlab based simulations, but I was difficult to install Python with all the dependencies in the computer lab. With shinylive the students will be able to run the simulations without installing anything at all, even in their phones. It's just great.
